src/realTimeReporter/reportingApi/attachData.ts

```diff
@@ -1,42 +1,71 @@
 // @ts-ignore
 import PublicReportingAPI from 'reportportal-client/lib/publicReportingAPI';
 import { LOG_LEVELS } from '../../constants';
-import { AttachmentRQ, Attribute, LogRQ } from '../../models';
+import { Attribute, LogRQ } from '../../models';
 
 type LogMessage = Pick<LogRQ, 'message'>;
 
+interface SendLogInterface {
+    info(message: LogMessage): void;
+    debug(message: LogMessage): void;
+    warn(message: LogMessage): void;
+    error(message: LogMessage): void;
+    trace(message: LogMessage): void;
+    fatal(message: LogMessage): void;
+}
+
 export interface AttachDataInterface {
     addAttributes(attributes: Array<Attribute>): void;
     addDescription(text: string): void;
-    addAttachment(level: Pick<LogRQ, 'level'>, file: Pick<AttachmentRQ, 'file'>, message: LogMessage): void;
-
-    addLog(level: Pick<LogRQ, 'level'>, message: LogMessage): void;
-    logInfo(message: LogMessage): void;
-    logDebug(message: LogMessage): void;
-    logError(message: LogMessage): void;
-    logTrace(message: LogMessage): void;
-    logFatal(message: LogMessage): void;
+
+    sendLog: SendLogInterface;
+    sendLaunchLog: SendLogInterface;
 }
 
 export const attachData: AttachDataInterface = {
     addAttributes: PublicReportingAPI.addAttributes,
     addDescription: PublicReportingAPI.addDescription,
-    addAttachment: PublicReportingAPI.addAttachment,
 
-    addLog: PublicReportingAPI.addLog,
-    logInfo: (message) => {
-        PublicReportingAPI.addLog(LOG_LEVELS.INFO, message);
-    },
-    logDebug: (message) => {
-        PublicReportingAPI.addLog(LOG_LEVELS.DEBUG, message);
+    sendLog: {
```

Comment:
`sendLog` expected to be a function, not an object. It seems to me we should think if it's really necessery to add an additional object for logging methods.

Reply:
Renamed to `log/launchLog`.
Readme file for the whole agent with two possible reporters will be updated in the scope of task for postFactum reporter.